ui/packages/catalog-ui-search/src/main/webapp/component/filter-comparator/filter-comparator.view.js

```diff
@@ -20,11 +20,15 @@ const template = require('./filter-comparator.hbs')
 const CustomElements = require('../../js/CustomElements.js')
 const metacardDefinitions = require('../singletons/metacard-definitions.js')
 
-var geometryComparators = ['INTERSECTS']
-var dateComparators = ['BEFORE', 'AFTER', 'RELATIVE', 'BETWEEN']
-var stringComparators = ['CONTAINS', 'MATCHCASE', '=', 'NEAR']
-var numberComparators = ['>', '<', '=', '>=', '<=']
-var booleanComparators = ['=']
+import {
+  geometryComparators,
+  geometryComparatorsAnyGeo,
+  dateComparators,
+  stringComparators,
+  stringComparatorsAnyText,
+  numberComparators,
+  booleanComparators,
+} from '../filter/comparators'
```

Comment:
I don't see this file anywhere in the PR. Was it excluded from the commit?

Reply:
The .tsx file should now be in the PR